### DISCUSSION TITLE
pdmodel转onnx模型报错Cannot found attribute beta in op: swish

### DISCUSSION THREAD

**Discussion by 666echo** [PaddlePaddle/PaddleOCR#12635] [Q&A] [answered]
paddlepaddle-gpu         0.0.0.post117
paddle2onnx              1.0.9
paddleocr                2.6

这是我使用的转换命令
paddle2onnx --model_dir ./output/rec_db_inference --model_filename inference.pdmodel --params_filename inference.pdiparams --save_file ./inference/rec_onnx/model.onnx --opset_version 10 --enable_onnx_checker True

下面是输出信息和报错
[Paddle2ONNX] Start to parse PaddlePaddle model...
[Paddle2ONNX] Model file path: ./output/rec_db_inference/inference.pdmodel
[Paddle2ONNX] Paramters file path: ./output/rec_db_inference/inference.pdiparams
[Paddle2ONNX] Start to parsing Paddle model...
[ERROR] Cannot found attribute beta in op: swish
已放弃 (核心已转储)

这个问题应该如何解决呢？

**Comment by justStarG** [comment]
Any solution?

**Comment by dencentding** [comment]
I have the same situation. Did you solve it?

**Comment by xbkaishui** [comment, marked as answer]
解决办法：
paddlepaddle-gpu 回退为2.5.2，重新导出模型，再转onnx正常。仅供参考

**Comment by AIwzx** [comment]
优秀

**Comment by 89080322** [comment]
> 解决办法： paddlepaddle-gpu 回退为2.5.2，重新导出模型，再转onnx正常。仅供参考
优秀,昨天想试试退版本..没执行力,拖拖拖,今天看到您说的  果然...
